CryptKeeperZK/crypt-keeper-extension
# feat: support expand mode

## Explanation

Details are below:
- [x] Support extension work in a new tab
- [x] Minor test mocks refactoring
- [x] Update onboarding password rules

## More Information

Closes #66

## Screenshots/Screencaps

![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/9adcba04-856d-4b4b-96da-1e30a3c6638a)
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/35d14507-33c5-450d-82a2-eb89bc7a537b)
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/4edfb8b2-0755-490d-bef1-0003d414ae66)

## Manual Testing Steps

1. Open extension popup
2. Open it in a new tab
3. Create new identity
4. Upload backup file
5. Try to open pages in separate tab and check the close icons (history)
6. Check onboarding backup upload
7. Check forgot password flow

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
